src/app/state/persist/syncTabs.ts

```diff
@@ -31,6 +31,7 @@ export function receiveInitialTabSyncState(
     ...prevState,
     theme: initialSyncState.theme,
     wallet: initialSyncState.wallet,
+    contacts: initialSyncState.contacts,
```

Comment:
As far as I remember this is cover byPlaywright syncTabs. Why it's not failing?

Reply:
Expanded those tests so they would fail https://github.com/oasisprotocol/oasis-wallet-web/pull/1741